golang/cosmos/Makefile

```
@@ -90,22 +91,35 @@ endif
 ###                           Protobuf                                    ###
 ###############################################################################
 
-proto-gen: proto-tools
-	PATH="${BIN}:$$PATH" ./scripts/protocgen.sh
+protoVer=0.15.3
+protoImageName=ghcr.io/cosmos/proto-builder:$(protoVer)
+protoImage=$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(protoImageName)
 
-proto-lint: proto-tools
-	${BIN}/buf lint
+proto-all: proto-format proto-lint proto-gen
```

`proto-format` appears to not exist.
```suggestion
proto-all: proto-lint proto-gen
```

---

and should `proto-all` encompass `proto-check-breaking` and/or `proto-update-deps`?

---

Good points. I'll add the `proto-format` target, since that's how cosmos-sdk does it, but I don't want to change cosmos-sdk's established `proto-*` target conventions.